packages/file_selector/file_selector_platform_interface/lib/src/platform_interface/file_selector_interface.dart

```diff
@@ -38,37 +38,37 @@ abstract class FileSelectorPlatform extends PlatformInterface {
   }
 
   /// Open file dialog for loading files and return a file path
```

Comment:
I think it's worth adding a comment that it returns null if the user cancel's the operation, to make it clear why it's nullable. (Same for `getSavePath` and `getDirectoryPath`.)

Reply:
Yaa sounds logical..